packages/@aws-cdk/aws-eks-v2-alpha/lib/cluster.ts

```diff
@@ -1535,7 +1480,7 @@ export class Cluster extends ClusterBase {
    * that manages it. If this cluster is not kubectl-enabled (i.e. uses the
    * stock `CfnCluster`), this is `undefined`.
    */
-  private readonly _clusterResource: ClusterResource;
+  private readonly _clusterResource: CfnCluster;
```

Comment:
I think we should update the command related to this property.

Reply:
Do you mean the comment above?